apps/codeforafrica/.env

```diff
@@ -8,3 +8,5 @@ GHOST_URL="https://longform.codeforafrica.org"
 GITHUB_BACKEND_REPO="codeforafrica/ui"
 GITHUB_BACKEND_BRANCH="main"
 GITHUB_AUTH_ENDPOINT="api/oauth/auth"
+GIT_REPO_DIRECTORY=../../
```

Comment:
Isn't this just equivalent to running the proxy server from root directory instead of app directory?

Reply:
I think it depends on how you run it for me I do
`pnpm --filter "./apps/codeforafrica" cms`
How do you run it from your side?

Reply:
That's equivalent to `cd apps/codeforafrica && pnpm cms`. What I'm saying (or thinking out loud) is why not just `pnpm cms` at root? Does the proxy server care from which app you'll be connecting from?

Reply:
Unless we make the install the server at the root folder. Right now only `codeforafrica` has the `cms`

Reply:
alternatively, we can just remove it and move the `NEXT_PUBLIC_APP_DIRECTORY` to `.env.local` @kilemensi

Reply:
Cool, then lets leave it in `codeforafrica` for now @esirK. We'll move install/move the proxy server to root when a 2nd app needs NetlifyCMS.